lib/features/authentication/ui/cubit/auth_cubit.dart

```diff
@@ -23,3 +23,22 @@ class AuthCubit extends Cubit<AuthState> {
     }
   }
 }
+
+// ================================================================================================================================
+
+class PasswordRecoveryCubit extends Cubit<PasswordRecoveryState> {
```

Comment:
Put in a separate file, the structure should be like this:

-features
    -authentication
        -ui
             -screens
                  -auth
                       -cubit
                            -auth_subit.dart
                            -auth_state
                       -auth_page.dart
                  -password_recovery
                      -cubit
                           - password_recovery_cubit.dart
                           -password_recovery_state.dart
                      -password_recovery_page.dart